File: 6. Count Smaller Than X - GFG/6.-count-smaller-than-x.c
// { Driver Code Starts
//Initial Template for C

#include<stdio.h>


 // } Driver Code Ends
//User function Template for C

// arr[]: input array
// n: size of array
// x: element for which you need to find smaller than x
int smallerThanX(int arr[], int n, int x)
{
    // your code here
    int count = 0;
    for (int i = 0; i < n; i++){
        if (arr[i] < x){
            count ++;
        }
    }
    return count;
}

// { Driver Code Starts.

int main() {
	int T;
	scanf("%d", &T);
	while(T--) 
	{
	    int N;
	    scanf("%d", &N);
	    int arr[N];
	    for(int i=0;i<N;i++)
	        scanf("%d", &arr[i]);
	    int x;
	    scanf("%d", &x);
	    printf("%d", smallerThanX(arr,N,x));
	    printf("\n");
	}
	return 0;
}  // } Driver Code Ends

File: Array/6. Count Smaller Than X - GFG/6.-count-smaller-than-x.py
#User function Template for python3

class Solution:
    def smallerThanX(self,arr,n,x):
        #return required ans
        #code here
        count = 0
        for i in range(n):
            if arr[i] < x:
                count += 1 
        return count

#{ 
#  Driver Code Starts
#Initial Template for Python 3

#contributed by RavinderSinghPB
if __name__ == '__main__':
    from math import inf
    t= int(input())
    
    for _ in range(t):
        n=int(input())
        arr=[int(e) for e in input().split()]
        x=int(input())
        ob=Solution()
        ans=ob.smallerThanX(arr,n,x)
        print(ans)
# } Driver Code Ends